.github/workflows/monitor-vms.yml

```diff
@@ -2,9 +2,8 @@
 name: Reboots VMs in a controlled way
 on:
   schedule:
-    - cron: /15 * * * *
```

Comment:
Let's please be more cautious and test thoroughly properly before checking in as Prod code :)